### PR TITLE
AGW: python: fix eventlet dependency

### PR DESCRIPTION
**Pull request by pshelar**
This patch fixes following issue, this is due to most recent release of eventlet.

======================================================================
ERROR: Failure: ImportError (cannot import name 'ALREADY_HANDLED')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vagrant/build/python/lib/python3.5/site-packages/nose/failure.py", line 39, in runTest
    raise self.exc_val.with_traceback(self.tb)
  File "/home/vagrant/build/python/lib/python3.5/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/vagrant/build/python/lib/python3.5/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/vagrant/build/python/lib/python3.5/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/vagrant/build/python/lib/python3.5/imp.py", line 234, in load_module
    return load_source(name, filename, file)
  File "/home/vagrant/build/python/lib/python3.5/imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 693, in _load
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 673, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/vagrant/magma/lte/gateway/python/magma/pipelined/openflow/tests/test_msg_hub.py", line 18, in <module>
    from magma.pipelined.openflow.messages import MessageHub
  File "/home/vagrant/magma/lte/gateway/python/magma/pipelined/openflow/messages.py", line 17, in <module>
    import ryu.base.app_manager  # pylint: disable=unused-import
  File "/home/vagrant/build/python/lib/python3.5/site-packages/ryu/base/app_manager.py", line 35, in <module>
    from ryu.app import wsgi
  File "/home/vagrant/build/python/lib/python3.5/site-packages/ryu/app/wsgi.py", line 109, in <module>
    class _AlreadyHandledResponse(Response):
  File "/home/vagrant/build/python/lib/python3.5/site-packages/ryu/app/wsgi.py", line 111, in _AlreadyHandledResponse
    from eventlet.wsgi import ALREADY_HANDLED
ImportError: cannot import name 'ALREADY_HANDLED'

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

## Test Plan
`make test` on debian.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by koolzz — APPROVED** [review]
thanks for the fix!

**Review by emakeev — APPROVED** [review]

**Review by themarwhal — APPROVED** [review]

**Comment by themarwhal** [comment]
we might want to backport this to a bunch of branches

**Review by tmdzk — APPROVED** [review]

File: lte/gateway/python/setup.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def os_release():
             'idna==2.8',
             'python-dateutil==2.8.1',
             'six>=1.12.0',
-            'eventlet>=0.24',
+            'eventlet==0.30.2',
```
Comment (ulaskozat) [inline lte/gateway/python/setup.py:128]: do we want `==` or `>=`?

File: lte/gateway/python/setup.py [hunk 2]
```diff
@@ -241,7 +241,7 @@ def os_release():
         'idna==2.8',
         'python-dateutil==2.8.1',
         'six>=1.12.0',
-        'eventlet>=0.24',
+        'eventlet==0.30.2',
```
Comment (ulaskozat) [inline lte/gateway/python/setup.py:244]: same comment.

File: lte/gateway/python/setup.py (hunk 1 above)
Comment (pshelar) [reply lte/gateway/python/setup.py:128]: pipelineD is having issue with 0.31. so the dependency is fixed at 0.30.2.

**Review by ardzoht — APPROVED** [review]

**Pull request merged**